src/pages/FormPage.spec.js

```diff
@@ -0,0 +1,58 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import FormPage from './FormPage'
+import { MemoryRouter } from 'react-router-dom'
+
+describe('FormPage', () => {
+  it('renders a form with a dropdown menu, a text input and date input', () => {
+    render(<FormPage onSubmit={jest.fn} />)
+
+    expect(screen.getByRole('combobox')).toBeInTheDocument()
+    expect(screen.getByRole('textbox')).toBeInTheDocument()
+    expect(screen.getByLabelText('Date')).toBeInTheDocument()
+  })
+  it('calls on correct action when submiting form', () => {
+    const submitForm = jest.fn()
+    render(
+      <MemoryRouter>
+        <FormPage onSubmit={submitForm} />
+      </MemoryRouter>
+    )
+
+    const buttons = screen.getAllByRole('button')
+    const submitButton = buttons[1]
+    userEvent.click(submitButton)
+
+    expect(submitForm).toHaveBeenCalledTimes(1)
```

Comment:
```suggestion
    expect(submitForm).toHaveBeenCalled()
```
this would do the same and is shorter